packages/cdk/lib/constructs/engines/engine.ts

```diff
@@ -14,4 +17,56 @@ export class Engine extends Construct {
     super(scope, id);
     this.logGroup = new LogGroup(this, "EngineLogGroup");
   }
+
+  protected toMountPoint(containerPath: string, volumeName: string): MountPoint {
+    return {
+      sourceVolume: volumeName,
+      containerPath: containerPath,
+      readOnly: false,
+    };
+  }
+
+  protected toVolume(fileSystem: FileSystem, accessPoint: AccessPoint, volumeName: string): Volume {
+    return {
+      name: volumeName,
+      efsVolumeConfiguration: {
+        fileSystemId: fileSystem.fileSystemId,
+        transitEncryption: "ENABLED",
+        authorizationConfig: {
+          accessPointId: accessPoint.accessPointId,
+          iam: "ENABLED",
+        },
+      },
+    };
+  }
+
+  protected createAccessPoint(fileSystem: FileSystem): AccessPoint {
+    return new AccessPoint(this, "AccessPoint", {
+      fileSystem: fileSystem,
+      posixUser: {
+        uid: "0",
+        gid: "0",
+      },
+    });
+  }
+
+  protected createAccessPointcreateFileSystem(vpc: IVpc): FileSystem {
```

Comment:
Should this just be `createFileSystem` It doesn't create an access point unless you meant to call that method internally?